Applications/SlicerApp/Testing/Python/SlicerMRBMultipleSaveRestoreLoopTest.py

```diff
@@ -91,19 +91,17 @@ def test_SlicerMRBMultipleSaveRestoreLoop(self):
     import SampleData
     mrHeadVolume = SampleData.downloadSample("MRHead")
 
-    # Place a fiducial
-    markupsLogic = slicer.modules.markups.logic()
+    # Place a control point
+    pointListNode = slicer.mrmlScene.AddNewNodeByClass("vtkMRMLMarkupsFiducialNode")
```

Comment:
This change is causing `py_SlicerMRBMultipleSaveRestoreLoopTest` to fail because it uses the `GetNodeTagName` and not the `GetDefaultNodeNamePrefix` so the added node name is "MarkupsFiducial" and not "F", but it is trying to get the node by doing `pointListNode = slicer.util.getNode('F')`

Reply:
Thank you for testing!
I fixed this issue but apparently I did not pushed that version. I've force pushed the updated version now.